src/Cameras/babylon.freeCamera.ts

```diff
@@ -15,14 +15,20 @@
         public inputs: FreeCameraInputsManager;
 
         //-- begin properties for backward compatibility for inputs
-        public get angularSensibility(): number {
+		/**
+		 * This setting affects input for the mouse.

```

Comment:
missing information as to how. Same goes to the setter (the parameter documentation would be the most important)